resources/views/partials/modules/footer.blade.php

```diff
@@ -1,14 +1,14 @@
 <footer class="py-4 leading-loose text-center lg:mt-32 bg-gray-50 md:leading-normal">
-    {{ __('Brought to you by the lovely folks at') }} <a href="https://tighten.co/" class="text-blue-800 border-hover">Tighten</a>.
+    {{ __('Brought to you by the lovely folks at') }} <a href="https://tighten.co/" class="border-hover">Tighten</a>.
```

Comment:
Were these bad?

Reply:
Nope! you told me to keep them black though 😁
Happy to turn them blue! Let me know

Reply:
Haha ok! I believe me.